projects/demo-maps/src/app/route-components/route-example-olperformance/route-map7.component.html

```diff
@@ -0,0 +1,18 @@
+<main class="content-area">
+    <ukis-map-ol [layersSvc]="layersSvc" [mapState]="mapStateSvc" id="olMap" [controls]="controls"></ukis-map-ol>
+</main>
+
+<clr-vertical-nav [clrVerticalNavCollapsible]="true" class="right" [clr-nav-level]="2">
+
+    <ukis-performance></ukis-performance>
```

Comment:
can you wrap this in a container so it can use the [layout of Clarity Vertical Nav](https://clarity.design/documentation/vertical-nav/nav-groups/normal/pidgey)

![image](https://user-images.githubusercontent.com/3969484/82050197-00cff900-96b8-11ea-9e77-76013edbdf4a.png)